### DISCUSSION TITLE
Youtube may ban accounts if they mass report comments

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#589] [General]
I tried to scan 5 videos from MrBeast, and found a LOT of comments, I'm going to report them but won't youtube ban some accounts if they mass report comments all at once thinking the account is a bot? I don't know if they will but I'm wondering.

**Comment by Firecul** [comment]
Surely they'd only lock out the API key if that was the case.

**Reply by KendallDoesCoding** [reply]
> Surely they'd only lock out the API key if that was the case.

ok..

**Comment by ThioJoe** [comment]
They wouldn't have added an API resource that lets you report 50 comments with a single call if they didn't intend for people to use it

**Reply by KendallDoesCoding** [reply]
Ok, didn't know about this.
